src/OrbitGl/GpuTrackTest.cpp

```diff
@@ -3,10 +3,19 @@
 // found in the LICENSE file.
 #include <gtest/gtest.h>
 
+#include "CaptureViewElementTester.h"
 #include "GpuTrack.h"
 
 using orbit_gl::MapGpuTimelineToTrackLabel;
 
+TEST(GpuTrack, CaptureViewElementWorksAsIntended) {
+  orbit_gl::CaptureViewElementTester tester;
+  GpuTrack track = GpuTrack(nullptr, nullptr, tester.GetViewport(), tester.GetLayout(), 0, nullptr,
+                            nullptr, nullptr, nullptr);
+  EXPECT_EQ(2ull, track.GetChildren().size());
```

Comment:
Nit: For readability, I'd simply use `2` here.

Reply:
And below.

Reply:
I'd like to stick with the correct type to not have implicit conversion here.